packages/mobile/src/mock/new-item.ts

```diff
@@ -23,3 +26,8 @@ export const CALLING_CODE = [
   { label: 'UA', value: 'UA' },
   { label: 'PL', value: 'PL' },
 ];
+
+export const CONDITION = [
+  { label: i18next.t('make_a_post.NEW'), value: Condition.NEW },
+  { label: i18next.t('make_a_post.USED'), value: Condition.USED },
+];
```

Comment:
Why is it in mock? This belongs to the picker

Reply:
corrected